resources/views/admin/table/_partials/config.blade.php

```diff
@@ -0,0 +1,17 @@
+<script>
```

Comment:
Maybe all of this php code can be passed as json, or if not json at least stored is some hidden input fields? That way we could separate the php from js. Checkout the JavaScript package that I added to Laradium.

With that, you could easily use `JavaScript::put($myData)` and use it in front-end like this - `Laradium.my_property`